project/src/main/webapp/script.js

```diff
@@ -11,54 +11,77 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-var namesCheckedOff;
+var namesCheckedOff;//counts number of profiles selected for group
 var loginUsername = "";
+var vgChatMade;//flag that checks if a chat has been made for the vg category
 
-function getChat() {
-    fetch('/data')
+/*Get actual chat messages*/
+function getChat(page) {
+    fetch(page)
     .then(response => response.json())
     .then((funFacts) => {
         const uname  = document.getElementById('username');
         uname.innerText = funFacts.username;
 
-        const myStrings = document.getElementById('quote-container');
+        const myStrings = document.getElementById('message-container');
         myStrings.innerHTML = '';
         funFacts.comments.forEach((line) => {
             console.log(line);
             // check list element inner text
             var liElement = createListElement(line);
-            // var n = liElement.innerText.includes('test-user');
-            // console.log(n);
-            // if(n){
-            //     liElement.style = 'float:right;';
-            // } else{
-            //     liElement.style = 'float:left;';
-            // }
             myStrings.appendChild(liElement);
         });
     });
 }
 
-function addTable() {
+/*Get profile names that belong to chat*/
+function getChatMemberProfiles(page) {
+    fetch(page)
+    .then(response => response.json())
+    .then((profileNames) => {
+        const myStrings = document.getElementById('chat-members');
+        myStrings.innerHTML = '';
+        var title = 'Chat Members:';
+        var liElement = createListElement(title);
+        liElement.style='list-style: none; font-size: 150%; font-weight: bold;';
```

Comment:
Optional: It'd be more reusable to create a CSS class with this style and set the class on the liElement.